chingsley/ireporter
# Ch endpoint feedbacks

#### What does this PR do?
This pull request with the implementation of the feedback I got on my [endpoints.](url) These feedbacks are mainly concerned with validating user inputs including name, email, phone numbers, etc.

#### Description of Task to be completed?
Improve the validator functions to handle more cases.

#### How should this be manually tested?
Check out the heroku link at https://ireporter-db.herokuapp.com
Then use the link to test on Postman.
